test/test_ops.py

```diff
@@ -1113,277 +1111,45 @@ def test_bbox_convert_jit(self):
         torch.testing.assert_close(scripted_cxcywh, box_cxcywh, rtol=0.0, atol=TOLERANCE)
 
 
-class BoxTestBase(ABC):
-    @abstractmethod
-    def _target_fn(self) -> Tuple[bool, Callable]:
-        pass
+def area_check(box, expected, tolerance=1e-4):
+    out = ops.box_area(box)
+    torch.testing.assert_close(out, expected, rtol=0.0, check_dtype=False, atol=tolerance)
```

Comment:
Ok. So It is necessary to set `check_dtype = False` here. Since the boxes would be casted to higher dtype.
You can try running this by simply putting `check_dtype=True` It will throw error. 

As the calculated value is 10000 for box_area which is int32. (torch will automatically use int32 or you can set that)
But there is a case where we use int64 boxes.  This will lead to dtype mismatch.

Same applies for float too.

Safer assumption is to ignore dtypes when we are parametrizing the tests.

Hence keeping the ignore_dtype to True.